src/Bicep.Core.IntegrationTests/Emit/TemplateEmitterTests.cs

```diff
@@ -67,6 +68,15 @@ public async Task ValidBicep_TemplateEmiterShouldProduceExpectedTemplate(DataSet
 
             // validate that the template is parseable by the deployment engine
             TemplateHelper.TemplateShouldBeValid(outputFile);
+
+            // validate source map
+            var actualSourceMapJson = JToken.FromObject(result.SourceMap!);
+
+            actualSourceMapJson.Should().EqualWithJsonDiffOutput(
+                TestContext,
+                JToken.Parse(dataSet.SourceMap!),
+                expectedLocation: DataSet.GetBaselineUpdatePath(dataSet, DataSet.TestFileMainSourceMap),
+                actualLocation: sourceMapFilePath);
```

Comment:
Please could we create a separate test for verifying the source map baselines are accurate?

Reply:
Sure.